.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Checklist
+
+* [ ] I have updated the end-to-end tests to work with my changes
+* [ ] I have updated the OpenAPI spec
```

Comment:
This shouldn't be the responsibility of PRs in this repo because the spec is in `opg-sirius`